docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Quickstart - Deploy a connected registry to a nested IoT Edge device
+description: Use Azure Container Registry CLI commands and Azure portal to deploy a connected registry to a mested Azure IoT Edge device.
+ms.topic: quickstart
+ms.date: 04/28/2021
+ms.author: memladen
+author: toddysm
+ms.custom:
+---
+
+# Quickstart: Deploy a connected registry to a nested IoT Edge device
+
+In this quickstart, you use [Azure Container Registry][container-registry-intro] commands to deploy a connected registry to a nested Azure IoT Edge device. You can review the [ACR connected registry introduction](intro-connected-registry.md) for details about the connected registry feature of Azure Container Registry.
+
+[!INCLUDE [quickstarts-free-trial-note](../../includes/quickstarts-free-trial-note.md)]
+
+[!INCLUDE [azure-cli-prepare-your-environment.md](../../includes/azure-cli-prepare-your-environment.md)]
+
+## Before you begin
+
+This tutorial requires a nested Azure IoT Edge device to be set up upfront. You can use the [Deploy your first IoT Edge module to a virtual Linux device](../iot-edge/quickstart-linux.md) quickstart guide to learn how to deploy a virtual IoT Edge device. You can also look at [Tutorial: Create a hierarchy of IoT Edge devices](../iot-edge/tutorial-nested-iot-edge.md) to learn how to configure hierarchical IoT edge devices. The connected registry is deployed as a module on the nested IoT Edge device.
+
+This tutorial also requires that you have set up a connected registry on a top level IoT Edge device by following the [Quickstart: Deploy a connected registry to an IoT Edge device](quickstart-deploy-connected-registry-iot-edge-cli.md).
+
+Also, make sure that you have created the connected registry resource in Azure as described in the [Create connected registry using the CLI][quickstart-connected-registry-cli] quickstart guide. Only `mirror` mode will work for this scenario.
+
+## Create a client token for access to the parent registry
```

Comment:
@toddysm when the customers use latest azure cli to create a nested connected registry, the command will automatically create a sync scope-map and token. It also updates the parent connected registry sync scope-map to include any new repos. So it seems there is no need to have this section to ask customers to create a sperate client token.

```
az acr connected-registry create --registry myacr --name myconnectedmirror --mode mirror --parent myconnectedregistry --repository "hello-world" "acr/connected-registry"
Command group 'acr connected-registry' is in preview and under development. Reference and support levels: https://aka.ms/CLI_refstatus
Adding 'myconnectedmirror' gateway permissions and repo(s) 'hello-world, acr/connected-registry' Mirror permissions to connected registry 'myconnectedregistry' sync scope map.
If sync scope map 'myconnectedmirror-sync-scope-map' already exists, its actions will be overwritten
If sync token 'myconnectedmirror-sync-token' already exists, it properties will be overwritten
```

@rosanch @jaysterp

Reply:
When the customers later deploy the nested connected registry in IoT device, they can use the auto-generated `myconnectedmirror-sync-token` as the credential to pull the image from parent connected registry.

Reply:
https://github.com/Azure/acr/blob/main/docs/preview/connected-registry/quickstart-connected-registry-cli.md needs to be updated to include the instruction to create `myconnectedmirror` with parent `myconnectedregistry `

Reply:
https://github.com/Azure/acr/blob/main/docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md has a similar section to create separate client token to access parent cloud registry. Similarly, it can be replaced with sync token credentail.

Reply:
> https://github.com/Azure/acr/blob/main/docs/preview/connected-registry/quickstart-connected-registry-cli.md needs to be updated to include the instruction to create `myconnectedmirror` with parent `myconnectedregistry `

> https://github.com/Azure/acr/blob/main/docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md has a similar section to create separate client token to access parent cloud registry. Similarly, it can be replaced with sync token credentail.

Can we file bugs for the above two for the other documents?

I removed the section from this document. However, I see one problem - now the sync token details are part of the connection string and we will be asking the customers to extract them from there.